22.RaytracedAO/CommandLineHandler.hpp

```diff
@@ -21,11 +21,16 @@ Description and usage:
 
 -SCENE:
 	some/path extra/path which will make it skip the file choose dialog
+
+-PROCESS_SENSORS ID:
+	It will control the behaviour of sensors in the app as detailed above.
+	If the option is not passed, then it defaults to RenderAllThenInteractive.
+	If the ID is not passed, then it defaults to 0.
 	
 Example Usages :
 	raytracedao.exe -SCENE=../../media/kitchen.zip scene.xml
 	raytracedao.exe -SCENE=../../media/kitchen.zip scene.xml -PROCESS_SENSORS RenderAllThenInteractive
```

Comment:
you want `RenderAllThenInteractive` to also take an ID

Reply:
It can, but it cannot also (in which case it defaults 0). This usage shows when `RenderAllThenInteractive` doesn't take an ID.